lib/Exporter/Writer/MimeTypedWriterInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+interface MimeTypedWriterInterface extends WriterInterface
+{
+    /**
+     * @return string the mime type of the output
+     */
+    public function getMimeType();
```

Comment:
`Default` would mean you can set another, which might not be true… let's keep it simple ok? `getMimeType` really means "get the mime type we feel is the most appropriate", so I'll just add that as a description for the method.